.gitignore

```diff
@@ -50,3 +50,9 @@ __pycache__
 
 # Ignore echidna artifacts
 crytic-export
+
+#goland
+.debug/op-node-local.run.xml
+.debug/op-batcher-local.run.xml
+.debug/op-proposer-local.run.xml
```

Comment:
@jurajpiar why don't we add `.vscode/launch.json` here?

Reply:
done